docs/content/Examples-Tutorials-Recipes/Multiple-Data-Sources-With-Same-Schema.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Manage Access to Multiple Data Sources
+permalink: /recipes/multiple-sources-same-schema
+category: Examples & Tutorials
+subCategory: Data sources
+menuOrder: 3
+---
+
+## Use case
+
+We need to access the data from different data sources for different tenants.
+For example, we are the platform for the online website builder, and each client
+can only view their data. The same data schema is used for all clients.
+
+## Configuration
+
+Each client has own database. To manage access to the databases, we need to use
+the
+[`contextToAppId`](https://cube.dev/docs/config#options-reference-context-to-app-id)
+function and the
+[`driverFactory`](https://cube.dev/docs/config#options-reference-driver-factory)
+property. [JSON Web Token](https://cube.dev/docs/security) includes information
+about the tenant name in the `tenant` property.
+
+```javascript
+const PostgresDriver = require('@cubejs-backend/postgres-driver');
+
+module.exports = {
+  contextToAppId: ({ securityContext }) =>
+    `CUBEJS_APP_${securityContext.tenant}`,
+
+  driverFactory: ({ securityContext } = {}) => {
+    if (securityContext.tenant === 'cubeDev') {
+      return new PostgresDriver({
+        database: 'cubeDev',
+        host: 'postgres',
+        user: 'postgres',
+        password: 'example',
+        port: '5432',
+      });
+    } else {
+      return new PostgresDriver({
+        database: 'ecom',
+        host: 'demo-db.cube.dev',
+        user: 'cube',
+        password: '12345',
+        port: '5432',
+      });
+    }
+  },
+};
+```
+
+## Query
+
+To get users for different tenants, we will send two identical requests with
+different JWTs:
```

Comment:
I think that it makes less sense to show the query here (it's the same for both tenants). I suggest showing the payloads of JWTs instead.

Reply:
Change to the JWT`s payload